### PR TITLE
Photo warning

### LINKED ISSUE #113: Add error notification if the photo has been dismissed
following #102 
reference #85

### PR DESCRIPTION
**Pull request by skjnldsv**
Fix #113

- [x] Validate function
- [x] Debug logs
- [x] Design
- [x] Testing
- [x] ~~Why is the groups test failing with the `console.debug`~~ Will see later

### CODE REVIEW

**Comment by irgendwie** [comment]
Works! The only problem that I have is, that it's not obvious, that the image got removed. Maybe we should information about that in the tool-tip from the little ! sign. For me that's the only obvious information source of failed properties (apart from the red mark of input fields).

**Comment by skjnldsv** [comment]
The image is removed on purpose. We can't guess what types the image is. So we have an unusable binary base64 string :/

File: js/models/contact_model.js [hunk 1]
```diff
@@ -339,10 +339,17 @@ angular.module('contactsApp')
...
-							this.failedProps.push(prop);
-							this.removeProperty('photo', property);
-							property = undefined;
-							console.debug(this.uid()+': Photo removed');
+							if (MimeService.b64mime(property.value) !== '') {
+								this.failedProps.push(prop);
+								property.meta.type = MimeService.b64mime(property.value);
+								console.debug(this.uid()+': Photo detected as ' + property.meta.type);
+							}
+							else {
```
Comment (skjnldsv) [inline js/models/contact_model.js]: please on the same line :p

File: js/models/contact_model.js [hunk 2]
```diff
@@ -307,6 +309,7 @@ angular.module('contactsApp')
 				return matchingProps.length > 0;
 			},
 
+			/* eslint-disable no-console */
```
Comment (Henni) [inline js/models/contact_model.js]: This, the two code comments (L321, L327) and the `console.debug` calls (L351, L356) should be removed.

Comment (Henni) [inline js/models/contact_model.js]: Ah, this seems to be intended. Should we hide this behind some flag?
(`debug` sounds a bit wrong, maybe `console.error`, `console.warning` or `console.info`might be better.)

Comment (skjnldsv) [inline js/models/contact_model.js]: Warning seems good to me :)

**Pull request merged**